.gitattributes

```diff
@@ -2,6 +2,7 @@
 *.txt text eol=lf
 langref.html.in text eol=lf
 deps/SoftFloat-3e/*.txt text eol=crlf
+huffman-text.input text eol=lf
```

Comment:
How about this instead?

```suggestion
lib/std/compress/deflate/testdata/* binary
```

Reply:
Will try that locally and verify that it'll work

Reply:
Confirmed to work, might still have an issue with the CI not re-checking-out the relevant file though :(